wagtail/core/models/copying.py

```diff
@@ -0,0 +1,96 @@
+from django.db import models
```

Comment:
Wonder if this should be moved to `wagtail/core/utils/copying.py` instead? Since there are no models here?

Reply:
I'm not really a fan of 'utils' modules - they tend to become a bit of a dumping ground. Moving it outside of core.models also raises questions about whether we expect this to be re-used in other apps/modules, and if so, whether they should become un-underscored... I think that's a yak-shave too far for this PR, and for now we should stick to just splitting up wagtail.core.models rather than actually moving things out of it.